### DISCUSSION TITLE
Possible to Connect Echelon Beat Advanced Heartrate Monitor (without Echelonfit Subscription)?

### DISCUSSION THREAD

**Discussion by ryegye24** [cagnulein/qdomyos-zwift#2393] [Q&A] [answered]
I've been using QZ with my Echelon EX5 for awhile and it's been working great. I just received an Echelon heart rate monitor, but I wasn't able to pair it directly to my tablet. Based on what I found in Echelon's instructions, the official way to pair it is through the Echelon app. 

Does anyone know if there's another way to pair it?  
Or if pairing it the official way will let it work in QZ? And if so, would it continue to work after I cancel the subscription (I'd need to start a trial subscription just to get into the app to do the original pairing)?

**Comment by cagnulein** [comment]
is it a bluetooth one?

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno lun 17 giu 2024 alle 19:18 ryegye24 ***@***.***> ha
scritto:

**Reply by ryegye24** [reply]
Yes it's a bluetooth heart rate monitor (sorry I should have mentioned that)

**Reply by cagnulein** [reply]
So do you see under the heart rate belt name setting in qz?

**Reply by ryegye24** [reply]
No, I see the bike (and most other bluetooth devices in range) but not the heart rate monitor. I do see the heart rate monitor when I navigate to the Android bluetooth "pair new device" screen, but attempting to pair it from there results in an immediate error about incorrect PIN or passkey.

**Comment by cagnulein** [comment]
I guess it's Bluetooth 2.0 not 4.0 that's the reason. qz is only compatible
with bluetooth 4.0 and above

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno lun 17 giu 2024 alle 21:44 ryegye24 ***@***.***> ha
scritto:

**Reply by ryegye24** [reply]
hmm the website for the monitor says it's bluetooth 4.0 compatible, I'm guessing it has something to do with Echelon's software, since the instructions do explicitly say you have to pair it from their app rather than Android's bluetooth settings

**Comment by ryegye24** [comment, marked as answer]
Update: I got it working! For anyone else who runs into this problem, here's how I got it working

1. Created an Echelon account/subscription (luckily the first month is free)
2. Installed the EchelonFit app on the device with QZ
3. Followed these instructions to connect the heart rate monitor from within the EchelonFit app https://echelonfituk.zendesk.com/hc/en-gb/articles/13083375143187-How-do-I-connect-to-a-heart-rate-monitor
4. Close the EchelonFit app and start QZ

At that point, when I went into the heart belt settings the Echelon heart rate monitor showed up in the list and was able to connect!

**Comment by cagnulein** [comment]
that's great!

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno lun 17 giu 2024 alle 23:30 ryegye24 ***@***.***> ha
scritto:
